Packs/MalwareInvestigationAndResponse/IncidentFields/incidentfield-Malware_Detailed_Investigation_Summary.json

```diff
@@ -0,0 +1,30 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Malware"
+    ],
+    "caseInsensitive": true,
+    "cliName": "malwaredetailedinvestigationsummary",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_malwaredetailedinvestigationsummary",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Malware Detailed Investigation Summary",
+    "neverSetAsRequired": false,
+    "openEnded": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "shortText",
```

Comment:
Should be Multi-Select no? Or Am I missing here something?

Reply:
No this is short text deliberately if you set the json to an array field it looks like 
 
![image](https://user-images.githubusercontent.com/51712181/169072144-679e8141-20d9-4cbb-8450-6c392adf56ec.png)